desci-server/src/services/manifestRepo.ts

```diff
@@ -103,130 +104,150 @@ export type ManifestActions =
+//           .filter(Boolean) as string[];
+//         if (componentEntries.length > 0) {
+//           logger.info({ action, componentEntries }, `DocumentUpdater::Delete Components`);
+//           handle.change(
+//             (document) => {
+//               for (const path of componentEntries) {
+//                 const deleteIdx = document.manifest.components.findIndex((c) => c.payload?.path === path);
+//                 logger.info({ path, deleteIdx }, `DocumentUpdater::Delete`);
+//                 if (deleteIdx !== -1) document.manifest.components.splice(deleteIdx, 1);
+//               }
+//             },
+//             { time: Date.now(), message: action.type },
+//           );
+//         }
+//         break;
+//       case 'Rename Component Path':
+//         const components = latestDocument.manifest.components.filter(
+//           (component) =>
+//             component.payload?.path?.startsWith(action.oldPath + '/') || component.payload?.path === action.oldPath,
+//         );
+//         if (components.length > 0) {
+//           handle.change(
+//             (document) => {
+//               const components = document.manifest.components.filter(
+//                 (component) =>
+//                   component.payload?.path.startsWith(action.oldPath + '/') ||
+//                   component.payload?.path === action.oldPath,
+//               );
+//               for (const component of components) {
+//                 component.payload.path = component.payload?.path.replace(action.oldPath, action.newPath);
+//               }
+//             },
+//             { time: Date.now(), message: action.type },
+//           );
+//         }
+//         break;
+//       case 'Update Component':
+//         handle.change(
+//           (document) => {
+//             updateManifestComponent(document, action.component, action.componentIndex);
+//           },
+//           { time: Date.now(), message: action.type },
+//         );
+//         break;
+//       case 'Assign Component Type':
+//         handle.change(
+//           (document) => {
+//             updateComponentTypeMap(document, action.component.payload?.path, action.componentTypeMap);
+//           },
+//           { time: Date.now(), message: action.type },
+//         );
+//         break;
+//       case 'Set Drive Clock':
+//         handle.change(
+//           (document) => {
+//             if (document.driveClock && document.driveClock === action.time) return; // Don't update if already the latest
+//             document.driveClock = action.time;
+//           },
+//           { time: Date.now(), message: action.type },
+//         );
+//         break;
+//       default:
+//         assertNever(action);
+//     }
+//     latestDocument = await handle.doc();
+//     const updatedHeads = getHeads(latestDocument);
+//     logger.info({ action, heads: updatedHeads }, `DocumentUpdater::Exit`);
+//     return latestDocument.manifest;
+//   };
+// };
 
```

Comment:
Maybe clean this file up instead? Can always get the old definition back through git if necessary. At the very least, add some context to why this is commented out and in which situation it's again relevant :pray: